test/unit/websocket/index.test.ts

```diff
@@ -64,7 +65,7 @@ describe('WebSocket Connection Tests', () => {
       const websocketModule = require('../../../src/websocket')
       websocketModule.activeConnections = 0
 
-      await onConnection(mockSocket)
+      await onConnection(mockSocket, { socket: mockSocket } as unknown as IncomingMessage)
```

Comment:
can we add some test case/s for ip restrictions?

Reply:
added